addons/payment/views/payment_provider_views.xml

```diff
@@ -95,10 +95,24 @@
                         <page string="Fees" name="fees" attrs="{'invisible': [('support_fees', '=', False)]}">
                             <group name="payment_fees">
                                 <field name="fees_active"/>
-                                <field name="fees_dom_fixed" attrs="{'invisible': [('fees_active', '=', False)]}"/>
-                                <field name="fees_dom_var" attrs="{'invisible': [('fees_active', '=', False)]}"/>
-                                <field name="fees_int_fixed" attrs="{'invisible': [('fees_active', '=', False)]}"/>
-                                <field name="fees_int_var" attrs="{'invisible': [('fees_active', '=', False)]}"/>
+                                <field name="fees_dom_fixed"
+                                       widget="monetary"
+                                       style="max-width: 5rem"
+                                       options="{'currency_field': 'main_currency_id'}"
+                                       attrs="{'invisible': [('fees_active', '=', False)]}"/>
+                                <field name="fees_dom_var"
+                                       widget="percentage"
```

Comment:
Using the percentage widget here makes `100` be encoded as `1.0`!

I'm not against using the widget, but we need to be consistent with the code and with what is already encoded in dbs.

Reply:
Oh... Should I then write an upgrade script? This version makes way more sense to me.

Reply:
Yes, either you drop the percentage widget (you could eventually copy some places where we place a % in the view without using the percentage widget), or you make a migration script and modify the fees computation to consider the change in the field values.

Reply:
Ok thanks. I definitely think we should use the widget. I updated the code and added an upgrade script for existing dbs